components/eventing-controller/cmd/eventing-controller/main.go

```diff
@@ -42,19 +42,32 @@ func main() {
 	// Set controller core logger.
 	ctrl.SetLogger(zapr.NewLogger(ctrLogger.WithContext().Desugar()))
 
-	// Add schemes.
```

Comment:
Changes in this file are just to add the AddToScheme call for JetStream within the same `if`.